wysadzulice/templates/index.html

```diff
@@ -1,14 +1,43 @@
+{% load staticfiles %}
```

Comment:
We use 2-space intendetion in JS, CSS and HTML - please conform this template to it.